pkg/event/controller.go

```diff
@@ -96,6 +100,12 @@ func (gen *generator) Add(infos ...Info) {
 			logger.V(3).Info("skipping event creation for resource without a name", "kind", info.Kind, "name", info.Name, "namespace", info.Namespace)
 			continue
 		}
+		for _, event := range gen.eventsApplied {
+			if info.Reason == Reason(event) {
```

Comment:
This is the only place where we are testing whether the values passed to the flag match the allowed values `PolicyViolation`, `PolicyApplied`, `PolicyError`, `PolicySkipped`, Is this OK?

Reply:
/cc @realshuting